doc/guides/V8-api-for-node.md

```diff
@@ -0,0 +1,210 @@
+## V8 API usage for Node.js
+
+v8 docs, not particularly useful:
+- https://v8.dev/docs
+
+Some API info at https://v8.dev/docs/embed
+
+- An isolate is a VM instance with its own heap.
+  Node has one isolate.
+  Can get from `Local<Object>->GetIsolate()`, `context->GetIsolate()`
```

Comment:
We probably don’t want to encourage the former because it’s a bit slow and discouraged by the V8 people. Usually, if we pass something around that gives us access to the `Isolate*`, it’s the `node::Environment` instance